examples/dt/perfscale/scalelab/dataplane.md

```diff
@@ -0,0 +1,51 @@
+# Configuring and deploying the dataplane
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully deployed
+- The EDPM nodes are pre-deployed with RHEL 9.4 and the keys are added with ctlplane ip
```

Comment:
The EDPM nodes are pre-deployed with RHEL 9.4 and the [**SSH public**] keys are added with ctlplane ip

I assume you meant the SSH keys.